src/main/java/blusunrize/immersiveengineering/common/blocks/ItemBlockIESlabs.java

```diff
@@ -32,24 +17,31 @@ public ItemBlockIESlabs(Block b)
 		super(b);
 	}
 
+	//TODO: review: BlockIESlab is now a `SlabBlock`, `IESlabTileEntity` gone
+	// 			(due to flattening I suppose).
+	//      Vanilla onItemUse will invoke tryPlace, which referrs to the Block,
+	//			which is the SlabBlock. So snow layer replacements etc should be
+	//		  handled.
+	//   -> Proposal would be: testing first without overridden `onItemUse`.

```

Comment:
I'd suggest completely removing this class, IIRC it's only necessary because we used the same block for multiple types of slab.

Reply:
Also here, marked for removal.